openlibrary/plugins/upstream/code.py

```diff
@@ -99,14 +101,14 @@ class merge_work(delegate.page):
     path = "/works/merge"
 
     def GET(self):
+        i = web.input(records='', comment=None, mrid=None)
```

Comment:
```suggestion
        i = web.input(records='', mrid=None)
```
`comment` doesn't appear to be used here.

Reply:
`records` is used.
`comments` should be removed.